medusa/server/web/core/base.py

```diff
@@ -111,6 +111,7 @@ def __init__(self, rh, filename):
             'newsBadge': '',
             'toolsBadge': '',
             'toolsBadgeClass': '',
+            'floatingMenuBadge': '',
```

Comment:
No new python vars please. This should be implemented client side.

Reply:
Do you mean 'reading' the badge info from the tools badge and recreating it?

Reply:
I should be more specific - most of the info needed to create a badge with the correct color is only server side (apart from the unread news count which is global (`app.UNREAD_NEWS`).

Reply:
I mean there should be no changes to the python code for any new ui changes. Everything should now be done client side. The badges and HTML that're currently handled server side like the ones above will all be removed very soon.